super_good-solidus_taxjar.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "solidus_core", ">= 2.4.0"
-  spec.add_dependency "solidus_support"
+  spec.add_dependency "solidus_support", ">= 0.9.0"
```

Comment:
This will also force us to drop support for Rails 5.2, which seems ok as it's also officially unsupported now

Reply:
Hmm, I should probably add a changelog entry for this, thanks for calling it out!